### DISCUSSION TITLE
Adding profile picture to classic theme

### DISCUSSION THREAD

**Discussion by MathiasVDA** [rendercv/rendercv#214] [Ideas]
Hello, 

I just want to share how I found a workarround to add my profile picture to the classic theme.

I added my profile.jpg picture in the root folder where the My_Name_CV.yaml file is located.

I added this in classic/Preamble.j2.tex at the end of the list of \usepackage
`
\usepackage{graphicx}
`

And this at the end of classic/Header.j2.tex
```
\section{About Me}
    \begin{twocolentry}
        {
            \begin{figure}
            \raggedleft
            \includegraphics[width=0.7\linewidth]{../profile.jpg}
            \end{figure}    
        }
        {Summary text about me}
    \end{twocolentry} 
```

And then I installed https://github.com/MiKTeX/miktex on my Windows machine so that I could run:

`rendercv render --use-local-latex-command lualatex .\My_Name_CV.yaml`

**Comment by sinaatalay** [comment]
Thank you! There is a PR for built-in profile picture support (#193), thanks to @kael-k. We will release that in the next version.

**Reply by MathiasVDA** [reply]
Yes, I saw it. Looks good but I was too impatient 😅

**Comment by sinaatalay** [comment]
Profile picture is supported in RenderCV
